README.md

```diff
@@ -1,6 +1,6 @@
 <!-- BEGIN TOP -->
```

Comment:
You should restore the original README, have a look to [the documentation](https://stackoverflow.com/questions/307579/how-do-i-copy-a-version-of-a-single-file-from-one-git-branch-to-another)